copier/config/factory.py

```diff
@@ -58,6 +70,10 @@ def make_config(
     answers_data = DEFAULT_DATA.copy()
     answers_data.update(load_answersfile_data(dst_path, answers_file))
     answers_data.update(data or {})
+
+    if "_min_copier_version" in answers_data:
+        verify_minimum_version(answers_data["_min_copier_version"])
```

Comment:
I don't think this parameter should be saved in the answers file.

The one that declares the minimal version should be always the template, not the user IMHO.

Am I right?

Reply:
Hmm I think you're right. Even when updating, this is covered by the verification of the min version in the template copier.yml, so we don't need to copy it in the answers file.